app/templates/zazurc.json

```diff
@@ -1,6 +1,7 @@
 {
   "hotkey": "alt+space",
   "theme": "tinytacoteam/zazu-light-theme",
+  "primaryMonitor": false,
```

Comment:
@blainesch I like `"position": "primary"` or `"position": "detect"`. I can implement that if you'd like.

@JeroenBoersma Monitor would work too, but what would "force" change?